ee-feature-pack/galleon-common/src/main/resources/layers/standalone/jdr/layer-spec.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="jdr">
+    <dependencies>
+        <layer name="base-server"/>
+        <layer name="secure-management" optional="true"/>
```

Comment:
Replace this by `management` once https://github.com/wildfly/wildfly-core/pull/4296 is merged

Reply:
It is enough with wildfly/wildfly-core#4295 merged.